packages/e2e/cypress/e2e/dashboard.cy.ts

```diff
@@ -34,22 +34,23 @@ describe('Dashboard', () => {
         cy.visit(`/projects/${SEED_PROJECT.project_uuid}/dashboards`);
 
         cy.contains('Create dashboard').click();
+        cy.findByLabelText('Name your dashboard').type('Title');
+        cy.findByLabelText('Dashboard description').type('Description');
+        cy.findByText('Create').click();
 
-        cy.findByText('Edit dashboard').click();
-
-        cy.findByText('Add tile').click();
+        cy.findAllByText('Add tile').click({ multiple: true });
         cy.findByText('Saved chart').click();
         cy.findByRole('dialog').findByPlaceholderText('Search...').click();
         cy.contains('How much revenue').click();
         cy.findByText('Add').click();
 
-        cy.findByText('Add tile').click();
+        cy.findAllByText('Add tile').click({ multiple: true });
         cy.findByText('Markdown').click();
         cy.findByLabelText('Title').type('Title');
         cy.get('textarea').type('Content');
         cy.findByText('Add').click();
 
-        cy.findByText('Add tile').click();
+        cy.findAllByText('Add tile').click({ multiple: true });
```

Comment:
what does the multiple do?

Reply:
since we have more than one button that says 'add tile' we need to precise that or test fails